npc/re/quests/quests_16_2_illusion.txt

```diff
@@ -4866,3 +4867,1809 @@ OnInit:
+		mes "[Archeologist Leiphen]";
+		mes "I will come back. For research and for you, Narin.";
+		next;
+		mes "[Narin]";
+		mes "Really... Really? Are you going to come back here and find me?";
+		next;
+		mes "[Narin]";
+		mes "I don't need to be ... alone...?";
+		next;
+		select( "I am not leave you alone.", "Don't worry.", "I promise.", "I will come back." );	// note: the selection doesn't change the dialog
+		mes "[Narin]";
+		mes ".........";
+		next;
+		mes "[Narin]";
+		mes "......Thanks...";
+		next;
+		mes "[Narin]";
+		mes "Thanks... Really...";
+		next;
+		delitem 25315, countitem(25315);	// Piece_Of_Shell
+		specialeffect2 EF_CIRCLEPOWER2;
+		specialeffect EF_CIRCLEPOWER2, AREA, "Archeologist Leiphen#illtur01";
+		sleep2 500;
+		emotion ET_SURPRISE, getnpcid(0,"Archeologist Leiphen#illtur01");
+		mes "[Archeologist Leiphen]";
+		mes "Hu? Huh? It's just gone. The " + getitemname(25315) + ".";
+		mes "Suddenly on my hand... like it melted away.";
+		next;
+		mes "[Narin]";
+		mes "It has my wishes. Now... It doesn't need to exist.";
+		npctalk "Archeologist Leiphen : What does it mean?", "Archeologist Leiphen#illtur01", bc_self;
+		next;
+		emotion ET_ANGER;
+		mes "[Narin]";
+		mes "...I, I dont know the specific... Just It might be.";
+		next;
+		mes "[Narin]";
+		mes "...Oh, right. Here... I give it to you. Here, you too professor.";
+		unittalk .@account_id, "" + .@player_name$ + " : What's this?", bc_self;
+		npctalk "Archeologist Leiphen : It... It's an " + getitemname(25271) + "!", "Archeologist Leiphen#illtur01", bc_self;
+		next;
+		npctalk "Narin : ...Is it? In fact, I had it because it was pretty.", "", bc_self;
+		mes "[Narin]";
+		mes "It's a sign of promise. I will try to remember two of you. So come... come and find me.";
+		mes "Tomorrow, And the day after tomorrow... I'll be waiting...  Bro, And professor.";
+		next;
+		mes "[Narin]";
+		mes "And other... you said you would help them. I'll show you how. How can you help...";
+		next;
+		mes "[Narin]";
+		mes "^0000CDListen to what they want. Bro Even if you look very trivial, do something they'll be pleased.^000000";
+		mes "Please... Maybe one day...";
+		next;
+		select("Don't worry. I will.");
+		mes "[Narin]";
+		mes ":)...";
+		next;
+		mes "[Narin]";
+		mes "I, I was able to stand alone so far, because I could meet you two someday?";
+		emotion ET_OK;
+		mes "I will... remember... Please... come back.";
+		next;
+		mes "[Archeologist Leiphen]";
+		mes "I will be here untill I found the secrets.";
+		mes "Don't worry. Me, and " + .@player_name$ + " Do not forget Narin.";
+		unittalk .@account_id, "" + .@player_name$ + " : We must remember.", bc_self;
+		ill_turtle = 11;
+		erasequest 16117;	// Continuous story 2
+		setquest 16105;	// A short break, and a promise
+		getitem 25271,5;	// IllusionStone
+		getexp 3600000,2400000;	// todo unknown exp (irowiki values)
```

Comment:
Remember iRO has increased values.
This is kRO experience using 100% exp manual, so, should be half of it
![image](https://user-images.githubusercontent.com/40366993/204569056-ccbb98f4-78c0-4b26-b1e8-9586c2192dff.png)